src/java.base/windows/native/libnet/ResolverConfigurationImpl.c

```diff
@@ -114,6 +187,7 @@ static int loadConfig(JNIEnv *env, char *sl, char *ns) {
     flags |= GAA_FLAG_SKIP_ANYCAST;
     flags |= GAA_FLAG_SKIP_MULTICAST;
     flags |= GAA_FLAG_SKIP_FRIENDLY_NAME;
+    flags |= GAA_FLAG_INCLUDE_ALL_INTERFACES;
```

Comment:
Is this needed? As far as I could tell, we only retrieve DNS addresses here, and we only want the DNS servers associated with usable interfaces.

Reply:
Yes I think so. This change doesn't provide any obvious benefit, so let's revert it.

Reply:
Done.